lib/elixir/pages/Typespecs.md

```diff
@@ -139,6 +139,13 @@ Specifications can be overloaded just like ordinary functions.
     @spec function(integer) :: atom
     @spec function(atom)    :: integer
 
+You can also name your arguments in a typespec using `arg_name :: arg_type` syntax.
+This is particularly useful as a way to differentiate multiple arguments
+of the same type (or multiple elements of the same type in a type def):
+
+    @spec new_date(year :: integer, month :: integer, day :: integer)
```

Comment:
Good point...I did indeed forget a return value :(.

I don't see naming the return value on `new_date` as being useful.  Can you come up with another example where naming the return value would be useful?  It would be nice to use an example that actually shows the usefulness of that.